src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIOffenderService.kt

```diff
@@ -86,4 +106,20 @@ class CommunityAPIOffenderService(
       .block()
       ?.staffIdentifier
   }
+
+  fun getResponsibleOfficer(crn: String): ResponsibleOfficer {
+    val offenderManagersPath = UriComponentsBuilder.fromPath(offenderManagersLocation)
```

Comment:
_nit_ Can we inline `offenderManagersLocation`? They won't ever change and should be used through a single code path anyway, right?
```suggestion
    val offenderManagersPath = UriComponentsBuilder.fromPath("/secure/offenders/crn/{crn}/allOffenderManagers")
```

Reply:
this follows existing patterns which group urls in config e.g. 

```
community-api:
  locations:
    sent-referral: "/secure/offenders/crn/{crn}/referral/start/context/{contextName}"
    ended-referral: "/secure/offenders/crn/{crn}/referral/end/context/{contextName}"
    book-appointment: "/secure/offenders/crn/{crn}/sentence/{sentenceId}/appointments/context/{contextName}"
    reschedule-appointment: "/secure/offenders/crn/{crn}/appointments/{appointmentId}/reschedule/context/{contextName}"
    appointment-outcome-request: "/secure/offenders/crn/{crn}/appointments/{appointmentId}/outcome/context/{contextName}"
    notification-request: "/secure/offenders/crn/{crn}/sentences/{sentenceId}/notifications/context/{contextName}"
    offender-access: "/secure/offenders/crn/{crn}/user/{username}/userAccess"
    managed-offenders: "/secure/staff/staffIdentifier/{staffIdentifier}/managedOffenders"
    staff-details: "/secure/staff/username/{username}"
```

or 

```
hmppsauth:
  api:
    locations:
      auth-user-groups: "/api/authuser/{username}/groups"
      auth-user-detail: "/api/authuser/{username}"
      user-email: "/api/user/{username}/email"
      user-detail: "/api/user/{username}"
 ```

Reply:
yes -- do we benefit from that setup?

Reply:
we can ask the team what they think. it doesn't bother me having to look them up, and i like having them all together. but i can see your point.